### ISSUE TITLE
Links to mailing list and chat don't work

### ISSUE THREAD

**Issue by apollo13** [canonical/vanilla-framework#5363]
The README.md has a link to freenode https://github.com/canonical/vanilla-framework/blob/main/README.md?plain=1#L6 and a mailing list https://github.com/canonical/vanilla-framework/blob/main/README.md?plain=1#L12 and none of them work.

Also the `devDependency Status` badge does not seem to work.

**Comment by bartaz** [comment]
Good catch, thanks! These are indeed quite outdated.

**Comment by jmuzina** [comment]
Thanks for the issue @apollo13 , this has been fixed by removing the dead links. 

Possibly, we may set up a space on the [Ubuntu Matrix instance](https://ubuntu.com/community/communications/matrix) - will update you if that happens :)

**Comment by apollo13** [comment]
Thank you @jmuzina. A chat channel somewhere would indeed be nice -- assuming vanilla is ment for public consumption as well. 👍

**Comment by jmuzina** [comment]
> Thank you [@jmuzina](https://github.com/jmuzina). A chat channel somewhere would indeed be nice -- assuming vanilla is ment for public consumption as well. 👍

@apollo13  Fully agree! It's not a question of wanting it, it's a question of setting up the instance (I don't myself have the permissions to create the matrix space :p ) - but I'll get that sorted out.

**Comment by apollo13** [comment]
@jmuzina Any luck with setting up a channel?

**Comment by jmuzina** [comment]
> [@jmuzina](https://github.com/jmuzina) Any luck with setting up a channel?

@apollo13 yes, we have a go-ahead for establishing a Matrix channel, I'll get one created & share it in a bit :)

**Comment by jmuzina** [comment]
@apollo13 The room has been created! https://matrix.to/#/#vanilla:ubuntu.com
